src/test/java/com/crud/tasks/config/AdminConfigTest.java

```diff
@@ -0,0 +1,24 @@
+package com.crud.tasks.config;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AdminConfigTest {
+
+    @Test
```

Comment:
Moim zdaniem ten test jest bez sensu. Parametry aplikacji sa czyms zewnetrznym dla niej i moga sue znieniac. jesli bym chcial zmienim maila, to musialbym zmieniac tez test.